### ISSUE TITLE
#64279 layered borg eyelights above mobs and cover-overlays

### ISSUE THREAD

**Issue by theOOZ** [tgstation/tgstation#64385]
#64279, @zxaber

## Reproduction:

Problem 1:
Mount a mob on your borg, or stack onto the same tile. The borg's eyelights will be above all mobs, even if the borg is at the bottom.

Problem 2:
Create an icon_state for the cover overlays (what shows up when you open a borg to remove cells etc.) that occupies the same space as eyelights would. Open the cover, you'll find the borg's eyelights above the cover overlay.

## Demo:
(This model is from a downstream, hence why problem 2)

https://user-images.githubusercontent.com/77534246/150835513-6ce3e0c1-6fb4-4314-8fc6-411826e68347.mp4

**Comment by zxaber** [comment]
Planes are the big dumb.

According to VV, mobs are on plane -3. I set the overlay plane to -2 and it started showing up again. I have no idea what the correct plane is.

Maybe I should be using layers, I dunno.

**Comment by theOOZ** [comment]
just found out it clips through tram's Z-level too.

![pic](https://puu.sh/IFA0J/9a2aa53e7c.jpg)

**Comment by timothymtorres** [comment]
Whoever added the felind borg probably didn't set them to the correct plane.  Do the other borg types have this problem as well?

**Comment by theOOZ** [comment]
> Whoever added the felind borg probably didn't set them to the correct plane. Do the other borg types have this problem as well?

Yes, this is happening on all our borgs since the PR mentioned in OP.
We also discovered that because the borg lights are now above the gameplane, they remain visible outside of FOV, too.
